src/wing/CTiglWingBuilder.cpp

```diff
@@ -213,6 +241,44 @@ PNamedShape CTiglWingBuilder::BuildShape()
 
     return loft;
 }
+
+// creates the inside and outside cap of the wing
+Standard_Boolean CreateSideCap(const TopoDS_Wire& W,
```

Comment:
Is this a duplicate of CTiglPatchShell::AddSideCap?

Reply:
Could be.  I did not add the function but I just moved the already existing code block into the #ifdef to improve code coverage ;)

Reply:
This code is now removed.